pkg/k8s/certificate.go

```diff
@@ -115,3 +232,24 @@ func SubmitCSR(ctx context.Context, config *cfg.Config, restClient *RestClient,
 	log.Infof("created CSR: %v", created)
 	return nil
 }
+
+func getKubernetesVersion(clientset kubernetes.Interface) (*VersionInfo, error) {
+	v, err := clientset.Discovery().ServerVersion()
+	if err != nil {
+		return nil, fmt.Errorf("failed to check k8s version: %v", err)
+	}
+
+	if v.Major != "1" {
+		return nil, nil
```

Comment:
I think you either want to return an err or proceed when version is higher than 1, so you can return the minor & major in the VersionInfo struct.